Content.Client/Paper/UI/PaperBoundUserInterface.cs

```diff
@@ -19,14 +19,21 @@ public PaperBoundUserInterface(ClientUserInterfaceComponent owner, Enum uiKey) :
         protected override void Open()
         {
             base.Open();
+            var entityMgr = IoCManager.Resolve<IEntityManager>();
+
             _window = new PaperWindow
             {
-                Title = IoCManager.Resolve<IEntityManager>().GetComponent<MetaDataComponent>(Owner.Owner).EntityName,
+                Title = entityMgr.GetComponent<MetaDataComponent>(Owner.Owner).EntityName,
             };
             _window.OnClose += Close;
             _window.Input.OnTextEntered += Input_OnTextEntered;
-            _window.OpenCentered();
 
+            if (entityMgr.TryGetComponent<PaperVisualsComponent>(Owner.Owner, out var visuals))
```

Comment:
*.Owner is deprecated if i recall correctly from mr sloth, pass the owner in via method arguments

Reply:
there isn't a replacement for this yet AFAIK.

Reply:
BUI's need a refactor sometime >.>